README.md

```diff
@@ -171,6 +180,33 @@ If none are returned for the `xConnect.client` certificate, you probably need to
 PS>  Grant-Permission -Identity sitecore -Permission GenericRead -Path 'cert:\localmachine\my\9CC4483261B92D7C5B32239115283933FC5014C4'
 ```
 
+## Certificates issues with Commerce
+Communication between services might fail when the certificates are not installed correctly. Verify what certificates are installed by:
+```
+PS> Get-ChildItem cert:\localmachine\my
+```
+
+The certificate thumbprint in `<Your>.Commerce.Engine/wwwroot/config.json` should match the one in `Y.Commerce.Engine/Sitecore.Commerce.Engine.Connect.config`.
+Note that when you have build new images the thumbprint in the `<Your>.Commerce.Engine/wwwroot/config.json` has to be manually updated.
+
+To determine and set the root certificate to use for a HTTPS connection:
+1. Determine certificate used for IIS: `PS> Get-WebBinding | Select certificateHash`
+2. Determine root certificate used to sign the in step 1 obtained certificate: `PS> Get-ChildItem cert:\localmachine\my\<certificateHash> | Select Issuer`
+3. Lookup thumbprint of the issuer: `PS> Get-ChildItem cert:\localmachine\root\
+3. Export the root certificate: `PS> Export-Certificate -Cert cert:\localmachine\root\<thumbprint> -FilePath <file>`
+4. Import the root certificate (on the client): `PS> Import-Certificate <file> -CertStoreLocation cert:\localmachine\root`
+
 ## Commerce setup
 - We have quite a lot of custom powershell scripts for trivial installation tasks. This is because the commerce SIF scripts contain hardcoded values. For example, it is not possible to use hostnames other than localhost. We should be able to remove this custom code when those scripts get fixed.
 - During the installation of the commerce server instances, it tries to set permissions on the log folder. For some reason, this results in an exception saying the access control list is not in canonical form. This can be ignored, because the log folders are mounted on the host. However, it does cause an annoying delay in the installation. 
+
+## Solr errors in Sitecore log
+After a clean start Sitecore reports errors like:
+```
+ERROR: [doc=sitecore://master/{731cb645-faa3-4440-9511-a27556a63ad9}?lang=fr-fr&amp;ver=1&amp;ndx=sitecore_master_index] unknown field '_indexname_t_fr'
+```
+
+Populating the Solr managed schemas will solve this, e.g. do this via the Sitecore Control Panel.
```

Comment:
Kunnen we dat ook niet automatiseren?

Reply:
Ja, maar ik wist ff niet zo snel hoe...

Reply:
Ah gevonden;
```
"UpdateSolrSchema": {
            // Update the solr schemas.
            "Type": "SitecoreUrl",
            "Params": {
                "SitecoreInstanceRoot": "[concat('http://', parameter('SiteName'))]",
                "SitecoreActionPath": "sitecore/admin/PopulateManagedSchema.aspx?indexes=all",
                "UserName": "admin",
                "Password": "[parameter('SitecoreAdminPassword')]"
            }
        }
```
Zal die nog ff toevoegen voor het begin van de SXA installatie.